chsdi/templates/htmlpopup/windatlas50.mako

```diff
@@ -0,0 +1,738 @@
+<%inherit file="base.mako"/>
+
+<%def name="table_body(c, lang)">
+  <%
+    baseUrl = request.registry.settings['api_url']
+  %>
+  <iframe src="${baseUrl}/rest/services/all/MapServer/${c['layerBodId']}/${c['id']}/extendedHtmlPopup?lang=${lang}" width="100%" height="400" frameborder="0"  style="border: 0"></iframe>
+</%def>
+
+<%def name="extended_info(c, lang)">
+
+<%
+  from chsdi.lib.raster.georaster import get_raster
+
+  r = get_raster('COMB')
```

Comment:
an iframe in the simple tooltip is a mess. how to solve this? we need an
expert tooltip

On Wed, Mar 9, 2016 at 6:05 PM, Gasser Loïc <notifications@github.com>
wrote:

> In chsdi/templates/htmlpopup/windatlas50.mako
> <https://github.com/geoadmin/mf-chsdi3/pull/1966#discussion_r55551853>:
>
> > @@ -0,0 +1,738 @@
> > +<%inherit file="base.mako"/>
> > +
> > +<%def name="table_body(c, lang)">
> > +  <%
> > +    baseUrl = request.registry.settings['api_url']
> > +  %>
> > +  <iframe src="${baseUrl}/rest/services/all/MapServer/${c['layerBodId']}/${c['id']}/extendedHtmlPopup?lang=${lang}" width="100%" height="400" frameborder="0"  style="border: 0"></iframe>
> > +</%def>
> > +
> > +<%def name="extended_info(c, lang)">
> > +
> > +<%
> > +  from chsdi.lib.raster.georaster import get_raster
> > +
> > +  r = get_raster('COMB')
>
> @gjn <https://github.com/gjn> @danduk82 <https://github.com/danduk82>
> This also means that we need to keep the DTM on non-legacy instances (as it
> is the case at the moment)
>
> —
> Reply to this email directly or view it on GitHub
> <https://github.com/geoadmin/mf-chsdi3/pull/1966/files#r55551853>.
>